core/src/main/java/org/apache/accumulo/core/spi/scan/DefaultScanServerSelector.java

```diff
@@ -110,14 +110,14 @@
  *       ]
  *     },
  *     {
- *       "scanTypeActivations":["slow"]
+ *       "scanTypeActivations":["slow"],
```

Comment:
I tried to copy this config, modify it, and use that.  However I found the example config had multiple json errors.